src/InterventionBackend.php

```diff
@@ -240,7 +248,9 @@ public function getImageResource()
         try {
             // write the file to a local path so we can extract exif data if it exists.
             // Currently exif data can only be read from file paths and not streams
-            $path = tempnam(TEMP_PATH, 'interventionimage_');
+            $tempPath = $this->config()->get('local_temp_path');
+            $tempPath = $tempPath !== null ? $tempPath : TEMP_PATH; 
```

Comment:
For future reference - we're accepting PHP7 syntax in minor branches now. You could have written `$tempPath = $this->config()->get('local_temp_path') ?? TEMP_PATH;`.

Reply:
I did that originally and then changed it to maintain compat as I thought it'd be required 😂